### ISSUE TITLE
Re-order pages in the Onboarding section

### ISSUE THREAD

**Issue by GBKS** [BitcoinDesign/Guide#550]
Per the discussion in #535.

Suggested order:
- First use (create a wallet)
- Funding a wallet
- Backing up a recovery phrase
   - Automatic cloud backup
   - Manual backup
- Protecting a wallet
- Restoring a wallet

Rationale is that for mainstream apps for daily use, some teams may decide that getting some amount of bitcoin into the wallet should happen before making users back up the keys (nothing to back up if users are not successful in funding the wallet).

Second part of this task is to revise the header images for all pages so they don't appear as a strict sequence (different projects may order things differently).

We can tackle this task once all pages in Onboarding are overhauled for Lightning.

**Comment by Bosch-0** [comment]
I can do this with my funding a wallet PR - do we have the header image source file anywhere? Happy to do that with the PR for all pages too :)

**Comment by GBKS** [comment]
Thanks for offering, I'd keep it as a separate task as to not overload your PR.

@sbddesign had some more specific thoughts on the headers. Don't remember who created the final images. I helped with the concept, which should still be a good start, you can find it [here](https://www.figma.com/file/qzvCvqhSRx3Jq8aywaSjlr/Bitcoin-Design-Guide-Illustrations-CO?node-id=1425%3A8176).

**Comment by Bosch-0** [comment]
This was addressed with https://github.com/BitcoinDesign/Guide/pull/604. Can open another issue for updating the onboarding banner images.
